recipes/pdfium/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class PdfiumConan(ConanFile):
+    name = "pdfium"
+    description = "PDF generation and rendering library."
+    license = "BSD-3-Clause"
+    topics = ("conan", "pdfium", "generate", "generation", "rendering", "pdf", "document", "print")
+    homepage = "https://opensource.google/projects/pdfium"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_libjpeg": ["libjpeg", "libjpeg-turbo"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_libjpeg": "libjpeg",
+    }
+
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package", "pkg_config"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("freetype/2.10.4")
+        self.requires("icu/69.1")
+        self.requires("lcms/2.11")
+        self.requires("openjpeg/2.4.0")
+        if self.options.with_libjpeg == "libjpeg":
+            self.requires("libjpeg/9d")
+        elif self.options.with_libjpeg == "libjpeg-turbo":
+            self.requires("libjpeg-turbo/2.1.0")
+
+    def build_requirements(self):
+        self.build_requires("pkgconf/1.7.4")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 14)
+        minimum_compiler_versions = {
+            "gcc": 8,
+            "Visual Studio": 15,
```

Comment:
No clang?

Reply:
clang builds fine with all compilers tested on c3i.
This is a filter for compilers that are causing problems.

Perhaps it should be given a better name or written differently.
Suggestions are welcome.

Reply:
Maybe a comment would be enough. This is the pattern we use to enforce a min CXX standard version and at a quick glance it looks like clang is missing.